cmd/glasskube/cmd/describe.go

```diff
@@ -329,6 +331,17 @@ func status(pkg ctrlpkg.Package) string {
 	}
 }
 
+func message(pkg ctrlpkg.Package) string {
+	pkgStatus := client.GetStatusOrPending(pkg)
+	if pkgStatus != nil {
+		if pkgStatus.Status == string(condition.Ready) {
```

Comment:
I think for the message it's not necessary to print it in green. Also, the message should always be printed, not only when the status is ready.

Reply:
I have removed the check for ready condition and also removed color formatting. Can you take a look and review @christophenne